src/tasks/AppleAppBuilder/Templates/runtime.h

```diff
@@ -4,6 +4,6 @@
 #ifndef runtime_h
 #define runtime_h
 
-void mono_ios_runtime_init (void);
+void mono_ios_runtime_init (int argc, char* argv[]);
```

Comment:
nit: aligning declaration and definition signatures
```suggestion
void mono_ios_runtime_init (int argc, char** argv);
```